LRQA-80963 CI Test with new method

JoyceWang08211 opened the pull request.

Kvale1733 committed:
LRQA-79873 Add target to terminate java process

Kvale1733 committed:
LRQA-79873 Add property to configure cluster jdbc_ping with tcp

Kvale1733 committed:
LRQA-79873 Add jdbc_ping with tcp xml to clustering dependencies

Kvale1733 committed:
LRQA-79873 Add StartupJDBCPINGWithZombieEntries test

seltong committed:
LPS-183792 Add getExternalReferenceCode to SystemObjectDefinitionManager interface

Commit:
LPS-185518 Create UpgradeJavaSetResultsSetTotalMethodCheck

Commit:
LPS-185518 Create Test UpgradeJavaSetResultsSetTotalMethodCheck

Commit:
LPS-179460 Add test to get aggregation of asset library vocabularies

Kvale1733 committed:
LRQA-80220 Open admin pages with url navigation

Kvale1733 committed:
LRQA-80220 Add widget with json

Commit:
LPS-187531 Create UpgradeJavaGetImagePreviewURLMethodCheck

achaparro committed:
LPS-185647 Add support to push and pop companies

achaparro committed:
LPS-185647 Important security check: since we are setting the CompanyThreadLocal based on the column company in the listener we need to be sure that the companyId column contains the correct company when the update/delete/insert operation was executed

Kvale1733 committed:
LRQA-80220 Replace UI navigation with URL navigation in Locale testcase

Kvale1733 committed:
LRQA-80220 Agree to terms and conditions with JSON

Kvale1733 committed:
LRQA-80220 Add page with JSON

Kvale1733 committed:
LRQA-80220 Set with JSON

Kvale1733 committed:
LRQA-80220 Replace UI navigation with URL navigation in GogoShell testcase

Kvale1733 committed:
LRQA-80220 Set with JSON

Kvale1733 committed:
LRQA-80220 Remove document in AWSStore test with JSON

Kvale1733 committed:
LRQA-80220 Remove document in Azure test with JSON

Kvale1733 committed:
LRQA-80220 Add instance with JSON

Kvale1733 committed:
LRQA-80220 Navigate to virtual instance admin with url

Kvale1733 committed:
LRQA-80220 Replace UI navigation with URL navigation in BundleBlacklist testcase

Kvale1733 committed:
LRQA-80220 Add comment to explain required restart for lpkg deployment

mbowerman committed:
LPS-185647 Allow another valid case here: The CompanyThreadLocal can be
set to SYSTEM while the object belongs to the default partition

Kvale1733 committed:
LRQA-80220 Blacklist bundles with osgi config

Kvale1733 committed:
LRQA-80220 Wait for console text to be present

Kvale1733 committed:
LRQA-80220 Remove redundant variable declaration

Kvale1733 committed:
LRQA-80220 Remove blacklist config while offline

Kvale1733 committed:
LRQA-80220 Change test names as UI is no longer used

Kvale1733 committed:
LRQA-80220 Use osgi config for ComponentBlacklist tests

Kvale1733 committed:
LRQA-80220 Assert console text before checking blacklist module in UI

Kvale1733 committed:
LRQA-80220 Assert reinstall as install was already logged

Kvale1733 committed:
LRQA-80220 Use URL navigation instead of UI for ComponentBlacklist testcase"

Kvale1733 committed:
LRQA-80220 Already covered by EnableBlacklistedComponentAtRuntime

Kvale1733 committed:
LRQA-80220 Add test property to blacklist component

Kvale1733 committed:
LRQA-80220 Add additional checks to assert component is blacklisted

Kvale1733 committed:
LRQA-80220 Go to systems settings with URL navigation

mbowerman committed:
LPS-185647 Add Listeners to set companyId as appropriate before and
after writing an entity to the database

mbowerman committed:
LPS-185647 Register the new listeners if db partitioning is enabled

mbowerman committed:
LPS-185647 Use append instead of set here since we are registering
POST_UPDATE EventListeners from multiple places now, and there is no
guarantee on which order they will be registered in

achaparro committed:
LPS-185647 In DB partition every partition has its own basic document type

Commit:
LPS-183792 Add validation for externalReferenceCode

Commit:
LPS-183792 Add new exception for object definition external reference code in service.xml

Commit:
LPS-183792 Add exceptions in ObjectDefinitionExternalReferenceCodeException

Commit:
LPS-183792 Add externalReferenceCode parameter to addSystemObjectDefinition method

achaparro committed:
LPS-185647 Let's execute the pop company after the pre event and before going to the database. This prevents having an incorrect companyThreadLocal when the post event is not executed if the SQL operations fails and I guess it also improves the performance.

istvansajtos committed:
LPS-188256 Improve memory handling

Commit:
LPS-183792 Override getExternalReferenceCode method for each System Object

Commit:
LPS-183792 Rename method

rafaellajordao committed:
LPS-189165 add 500 error utility page type to addUtilityPage macro

Commit:
LPS-183792 Add integration test

mbowerman committed:
LPS-185647 Push and pop need to remember the previous value that the
ThreadLocal had, so that it can be restored on pop.

In theory we could just convert the _companyIds to a LinkedList instead
of adding an entirely new LinkedList ThreadLocal, but I didn't do this
because:
* It would make the code more verbose everywhere in this class except
  for push and pop.
* The LinkedList is only relevant in partitioning is enabled (at least
  for now). I didn't want to introduce unnecessary LinkedList operations
  to happen all the time when partitioning is disabled.

istvansajtos committed:
LPS-188256 Remove unused property

Kvale1733 committed:
LRQA-80220 Blacklist bundle with test property

p2kmgcl committed:
LPS-188301 Migrate AddLayout to typescript

achaparro committed:
LPS-185647 DB Partition: Add upgrade to populate the companyId for DLFileEntryType

achaparro committed:
LPS-185647 Rename

p2kmgcl committed:
LPS-188301 Extract EditLayout

achaparro committed:
LPS-185647 I don't think we need a list here since we depend on boolean (pushed) that changes with every push/pop. Mixing preparedStatements that read the previousCompanyId in the incorrect order is unlikely but, if it happens, will generate an incorrect companyId in both cases, with list and without"

p2kmgcl committed:
LPS-188301 Add checkFriendlyURL

p2kmgcl committed:
LPS-188301 Build types

Commit:
LPS-183792 buildService

achaparro committed:
LPS-185647 With currentCompanyId we never return SYSTEM as defaultCompany in the CompanyThreadLocal. However we have to cover the opposite use case when the shardedModel is SYSTEM as it happens with portalPreferences loaded on the fly

achaparro committed:
LPS-185647 More information in the exception

achaparro committed:
LPS-185647 Adding tests

achaparro committed:
LPS-185647 SF

lfbesada committed:
LPS-188303 Adds a resource command to get the friendlyURL warning message when applies

lfbesada committed:
LPS-188303 Provides a context for the AddLayout React component

lfbesada committed:
LPS-188303 Add an util method to set the friendly URL warning message on session when applies

lfbesada committed:
LPS-188303 Add the friendly URL warning message if applies when we are adding or updating a page

lfbesada committed:
LPS-188303 Displays the friendly URL warning message if present

lfbesada committed:
LPS-188303 Adds the _requestProcessedWarning suffix in order to be handled by the LayoutCommonTag when creating a collection page

lfbesada committed:
LPS-188303 Displays a detailed warning message when editing the URL of a layout that already has conflicts

lfbesada committed:
LPS-188303 Adds language keys

lfbesada committed:
LPS-188303 BuildLang

antonio-ortega committed:
LPS-188246 Add alternative text to image viewer for those cases when we are using a Carousel template

istvansajtos committed:
LPS-188256 semver

gabrielwas committed:
LPS-188894 Support object actions inside SystemObjectDefinitionManager

gabrielwas committed:
LPS-188894 Create object actions while creating unmodifiable system objects

gabrielwas committed:
LPS-188894 Integration test

CalumR23 committed:
POSHI-564 Add new method to assert if an element is focused

markocikos committed:
LPS-188834 Add missing lang key

markocikos committed:
LPS-188834 Hide DS view selector if FF is not enabled

markocikos committed:
LPS-188834 Hide DS fragment if FF is not enabled

Kvale1733 committed:
LRQA-80220 Remove redundant test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 3 jobs passed in 21 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4652">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1407401_3735</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/71b8416aa3970501a39c7242e26d6c7d8948b846">71b8416aa3970501a39c7242e26d6c7d8948b846</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f91568295e98592365a19124becd9df4d09b3779">f91568295e98592365a19124becd9df4d09b3779</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2362529542">EE Development Acceptance (master) - 4651 - 2023-06-27[04:36:07]</a></p><details><summary><strong>ci:test:security - 0 out of 3 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/3735/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608391/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608390/"><strike>functional-tomcat90-mysql57-jdk8/0/1</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3735/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608391//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608391//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredByWebId</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3735/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3735/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredByWebId/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3735/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608390//consoleText">functional-tomcat90-mysql57-jdk8/0/1</a><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608390//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredToBlockImageSource</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3735/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3735/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3735/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/3735/">test-portal-acceptance-pullrequest(master)#3735</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3735/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>security<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/194">JoyceWang08211#194</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2363122206">[master] ci:test:security - JoyceWang08211 &gt; JoyceWang08211 - PR#194 - 2023-06-27[19:21:54]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/6856/">publish-testray-report#6856</a><br/></div></div>

JoyceWang08211 commented:
ci:test:security:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:security - 0 out of 2 jobs passed in 27 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/71b8416aa3970501a39c7242e26d6c7d8948b846">71b8416aa3970501a39c7242e26d6c7d8948b846</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/71b8416aa3970501a39c7242e26d6c7d8948b846">71b8416aa3970501a39c7242e26d6c7d8948b846</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2363320522">EE Development Acceptance (master) - 4652 - 2023-06-27[16:50:51]</a></p><details><summary><strong>ci:test:security - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/3738/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608493/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3738/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608493//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/608493//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource_">LocalFile.ContentSecurityPolicy#VerifyCSPCouldConfiguredToBlockImageSource</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3738/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3738/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentSecurityPolicy_VerifyCSPCouldConfiguredToBlockImageSource/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3738/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/3738/">test-portal-acceptance-pullrequest(master)#3738</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3738/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>security<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/194">JoyceWang08211#194</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2363485954">[master] ci:test:security - JoyceWang08211 &gt; JoyceWang08211 - PR#194 - 2023-06-27[23:25:19]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/6863/">publish-testray-report#6863</a><br/></div></div>

achaparro committed:
LPS-185647 If we do not throwException the resourcePermission should be always persisted in the database

Commit:
LPS-180004 Add test to get structured content with restricted fields by version

Commit:
LPS-180005 Add test to get structured content with only id field by version

Commit:
LPS-180006 Add test to get structured content versions with only id field

Commit:
LPS-180008 Add test to get aggregation of structured contents

victorg1991 committed:
LPS-186368 Update tests to take into account actions

Commit:
LPS-180008 Fix typo

Commit:
LPS-179460 Fix typo since the test should be to get aggregation of vocabularies

peerkar committed:
LPS-188222 portal-search-elasticsearch7-impl: Update index aliases gracefully in case company table is not in sync with the current ES mode and cluster state

peerkar committed:
LPS-183485 portal-search-rest-impl: Align schema tighter with other headless APIs

peerkar committed:
LPS-183485: gw buildRest

peerkar committed:
LPS-183485 portal-search-rest-impl: New dependencies

peerkar committed:
LPS-183485 portal-search-rest-impl: Add search result entity model

peerkar committed:
LPS-185905 portal-vulcan-api: Expose constructor to make Page class extensible

peerkar committed:
LPS-183485 portal-search-rest-impl: Add the extended Vulcan Page, SearchPage

peerkar committed:
LPS-183485 portal-search-rest-impl: Use FacetConfiguration, add vocabulary facet

peerkar committed:
LPS-183485 portal-search-rest-impl: Rename, vocabulary facet

peerkar committed:
LPS-183485 portal-search-rest-impl: Restore public getter names required for the desired kind of DTO conversion

peerkar committed:
LPS-183485 portal-search-rest-impl: Search result resource implementation

peerkar committed:
LPS-183485 portal-search-rest-test: Update tests and disable for now (LPS-186696)

peerkar committed:
LPS-183485 portal-language-lang: New language keys

peerkar committed:
LPS-183485 portal-language-lang: gw buildLang

Mylena-Rodrigues committed:
LPS-189094 Is not possible a Staff user ask for more info or reject a request

Commit:
LPS-179460 Fix typo

Mylena-Rodrigues committed:
LPS-189097 Fix Phone Number

Mylena-Rodrigues committed:
LPS-189097 Edit Request Form Page Name

Mylena-Rodrigues committed:
LPS-189097 Fix manager email address field required

Commit:
LRAC-14071 Add APIToken#ErrorMessageAppearsWhenAPITokenIsRevokedOrNonexistent test

Commit:
LPS-180008 Replace String.valueOf

Commit:
LPS-180008 Use for loop

Stephy6 committed:
LPS-186527 Create item selector api for asset tags

Stephy6 committed:
LPS-186527 Create item selector implementation for asset tags by adding the CriterionHandler class

Stephy6 committed:
LPS-186527 Add itemSelectorView and itemSelectorViewDescriptor for tags implementation

Stephy6 committed:
LPS-186527 Add TableItemView and ItemDescriptor for tags implementation

mbowerman committed:
LPS-185647 Make PMD stop complaining about "Do not call Assert#fail
improperly", although in my opinion this is less readable/intuitive than
what we had before.

CalumR23 committed:
POSHI-564 Update com.liferay.poshi.core to 1.0.123

Stephy6 committed:
LPS-186527 Add context classes and constants for tags item selector

CalumR23 committed:
POSHI-564 Add additonal method to check if element is focused

CalumR23 committed:
POSHI-564 Create new condtion and utilize it in new methods

kevhlee committed:
LPS-185518 Auto SF

kevhlee committed:
LPS-185518 Use String.startsWith() as the generic type argument could be mistakenly matched

kevhlee committed:
LPS-185518 SF, as used

kevhlee committed:
LPS-185518 Already trimmed

kevhlee committed:
LPS-185518 Use the regex to match the full method

kevhlee committed:
LPS-185518 Generated

peerkar committed:
LPS-188222 portal-search: Initialize the company index in concurrent mode too, because it won't necessarily exist when switching the mode / cluster

rafaellajordao committed:
LPS-187445 add LoadMoreResultsInPageTreeWhenEditContentPage test

rafaellajordao committed:
LPS-178475 add ViewEllipsisIconWhenHoverOnBodyOfWidget test

Commit:
LPS-183792 Update bookmarks erc

kresimir-coko committed:
LPS-188645 Hide FDSEntry sorting and filtering behind an FF

peerkar committed:
LPS-183485 portal-search-rest-impl: Rename (https://github.com/brianchandotcom/liferay-portal/pull/137077#issuecomment-1612771915)

Commit:
LPS-183792 Add ObjectDefinitionExternalReferenceCodeExceptionMapper

victorg1991 committed:
LPS-187134 Convert to Release FF

peerkar committed:
LPS-183485 portal-search-rest-impl: Reorder (https://github.com/brianchandotcom/liferay-portal/pull/137077#issuecomment-1612771915)

ealonso committed:
LPS-180008 Inline

ealonso committed:
LPS-180008 Uses getInt method in jsonObject

ealonso committed:
LPS-180008 SF

ealonso committed:
LPS-179460 SF

ealonso committed:
LPS-179460 Inline

ealonso committed:
LPS-179460 Uses getInt method in jsonObject

Commit:
LPS-189176 Add label to creationMenu entry

Commit:
LPS-189176 Pass emptyState props to FDS

ealonso committed:
LPS-186527 Adds correct version

ealonso committed:
LPS-186527 It should be isMultiple

ealonso committed:
LPS-186527 Make it consistent

ealonso committed:
LPS-186527 Removes unnecessary css classes and inline it

ealonso committed:
LPS-186527 Alphabetic order

ealonso committed:
LPS-186527 It is no needed

ealonso committed:
LPS-186527 group.getDescriptiveName(_themeDisplay.getLocale()) is returning the translated value, so it is no needed

ealonso committed:
LPS-186527 No needed

ealonso committed:
LPS-186527 Uses criterion method instead of creating an unnecessary one

ealonso committed:
LPS-186527 Reuse locale instead of using themeDisplay

ealonso committed:
LPS-186527 We should return always the same payload and adapt the frontend part accordingly

ealonso committed:
LPS-186527 Unused dependency

ealonso committed:
LPS-186527 We can just use the locale

ealonso committed:
LPS-186527 We should add the row checker when is multi selection, also the value of rowChecker was always false

ealonso committed:
LPS-186527 Adds diamond operator

ealonso committed:
LPS-186527 Alphabetic order

Commit:
LPS-189176 Build languages

ealonso committed:
LPS-186527 Use criterion groupIds

fmarcio committed:
LRAC-14149 Create lang

fmarcio committed:
LRAC-14149 Add ternary on pageDescription prop

fmarcio committed:
LRAC-14149 Update snapshot

ealonso committed:
LPS-186527 Simplify logic

samuelleolopes committed:
LPS-188984 Updated Product Version State from the drop-down menu

samuelleolopes committed:
LPS-188986 Updated key type State from the drop-down menu

nicolas-sss committed:
LPS-189260 Update imports.txt

mbowerman committed:
LPS-185647 Remove this case as it is known to fail on some environments.
The reason it occasionally fails has been investigated thoroughly and
appears to be because the enabling of partitioning in the test setup is
not working properly. Alberto and I have not been able to reproduce this
failure locally, but QA seems to reproduce it consistently so we are
removing it for now until we can get the test setup working correctly
consistently.

Commit:
LPS-180678 Add PagesTreeWithXSS#AccessToPageTreeWherePageNameWithXSS test

mbowerman committed:
LPS-185647 Semantic versioning

ealonso committed:
LPS-189165 Removes feature flag LPS-165914

CalumR23 committed:
POSHI-578 Do not generate if property is not set

CalumR23 committed:
POSHI-578 Remove attribute to support property value

interaminense committed:
LRAC-14113 Fix loading within ListComponent

interaminense committed:
LRAC-14113 Regen snapshots

interaminense committed:
LRAC-13485 Follow Up Master

Commit:
LPS-189285 Add property to not run the test on virtual instance

Commit:
LPS-189165 Modify cases to adapt new behavior

Commit:
LPS-178475 Reword

brianchandotcom committed:
POSHI-564 Rename

brianchandotcom committed:
POSHI-564 Typo?

brianchandotcom committed:
POSHI-564 Is this more accurate?

brianchandotcom committed:
POSHI-564 prep next

Commit:
LPS-189165 Remove summary temporarily

brianchandotcom committed:
LPS-188301 and LPS-188303 Wordsmith

brianchandotcom committed:
LPS-188301 SF

brianchandotcom committed:
LPS-188301 Regen

brianchandotcom committed:
LPS-186527 SF

brianchandotcom committed:
LPS-183792 try/finally

brianchandotcom committed:
LPS-183792 SF

brianchandotcom committed:
LPS-183792 SF

brianchandotcom committed:
LPS-183792 SF

brianchandotcom committed:
LPS-183792 We only have this for now

brianchandotcom committed:
LPS-183792 auto SF

brianchandotcom committed:
LPS-183792 Rename MSOD_ to L_. As long as our own modifiable system object definitions and unmodifiable system object definitions don't clash, we're fine.

brianchandotcom committed:
LPS-185647 and LPS-184771 Wordsmith

brianchandotcom committed:
LPS-183485 Sort

brianchandotcom committed:
LPS-183485 SF

Commit:
LPS-180004 Remove cases covered by integration tests

brianchandotcom committed:
LPS-183485 Sort

brianchandotcom committed:
LPS-183485 SF

brianchandotcom committed:
LPS-183485 SF

brianchandotcom committed:
LPS-183485 SF

brianchandotcom committed:
LPS-183485 auto SF

brianchandotcom committed:
LPS-183485 SF

brianchandotcom committed:
LPS-183485 SF

brianchandotcom committed:
LPS-183485 Only used once

brianchandotcom committed:
LPS-183485 Only used once

brianchandotcom committed:
LPS-183485 Inline

brianchandotcom committed:
LPS-187134 Wordsmith

marco-leo committed:
LPS-189312 add ancestors sites

juanjofgliferay committed:
LPS-188818 - Feat. Update fdsView infor after saving tab changes

ealonso committed:
LPS-189312 Simplify

dsanz committed:
LPS-188645 SF

Commit:
LPS-188922 Intercept request to check loading message

liferay-translation committed:
LPS-77699 Update Translations

juanjofgliferay committed:
LPS-188818 - Refactor. Remove types and reword variable

pedroliveirav committed:
LPS-189249 Show quantity column and adjust instance size column display logic

marcosapmf committed:
LRAC-14157 import com.liferay.portal.kernel.scheduler into bnd file

Commit:
LPS-179425 Remove cases covered by integration tests

lfbesada committed:
LPS-189243 If present use the parameter value instead of the persisted settings

lfbesada committed:
LPS-189243 When an error occurs then adds the parameters to the redirect URL

lfbesada committed:
LPS-189243 Extract common code to a base class

interaminense committed:
LRAC-14113 It is missing to return component

interaminense committed:
LRAC-14113 Regen snapshot

lfbesada committed:
LPS-189243 Remove the parameters when the request is processed success. In this way, we are preventing displaying an API Key that failed on validation in a previous form submission

brianchandotcom committed:
LRAC-13485 SF

brianchandotcom committed:
LPS-188222 SF

Commit:
LPS-188645 Add FF to poshi tests related to filter tabs e sort

kenjiheigel committed:
LRCI-970 Set test.url correctly for new virtual instance property

CalumR23 committed:
POSHI-580 Cast to string if paramter type requires String

kenjiheigel committed:
POSHI-580 SF

joelrocha68 committed:
LPS-189375 Fix viewStatus

joelrocha68 committed:
LPS-189375 Fix goToNavItem macro

thektan committed:
LPS-189384 frontend-taglib-clay: gradle baseline

CalumR23 committed:
POSHI-580 Add null check into if statement

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-187531 Create Test UpgradeJavaGetImagePreviewURLMethodCheck

Commit:
LPS-187531 Generated

brianchandotcom committed:
LPS-188834 regen

brianchandotcom committed:
POSHI-580 SF

brianchandotcom committed:
POSHI-580 prep next

JoyceWang08211 committed:
LRQA-80963 CI Test
